e2e-tests/portal/cypress/integration/journeys/maker/submit-a-deal-for-review/dealWithIncompleteAbout.json

```diff
@@ -17,6 +17,7 @@
   },
     "eligibility": {
       "status": "Completed",
+      "version": 4,
```

Comment:
shouldn't this be 5?

Reply:
the other files in this test have version 4, so kept it at 4.  Can change to 5

Reply:
v5 please

Reply:
done and changed mock data loader to v5